extensions/aerosol_optics/mo_aerosol_optics.F90

```diff
@@ -0,0 +1,633 @@
+! This code is part of Radiative Transfer for Energetics (RTE)
+!
+! Contacts: Robert Pincus and Eli Mlawer
+! email:  rrtmgp@aer.com
+!
+! Copyright 2015-2018,  Atmospheric and Environmental Research and
+! Regents of the University of Colorado.  All right reserved.
+!
+! Use and duplication is permitted under the terms of the
+!    BSD 3-clause license, see http://opensource.org/licenses/BSD-3-Clause
+! -------------------------------------------------------------------------------------------------
+! Provides aerosol optical properties as a function of effective radius for the RRTMGP bands
+!   Based on climatoligical aerosol optical properties used in MERRA2 as derived from
+!     the GOCART model for 15 aerosol types, including five dust types, five sea salt types,
+!     one sulfate type, and both hydrophobic and hydrophilic black carbon and organic carbon.
+!   Input aerosol optical data are stored in look-up tables.
+!
+!   References for gocart interactive aerosols:                        
```

Comment:
@mjiacono Please provide DOIs

Reply:
@RobertPincus Please clarify “DOI”

Reply:
DOIs are Document Object Identifiers, easily translatable into a precise citation. For Chin, for example, the DOI is 10.1029/2000JD900384, and you can access the paper as http://doi.org/10.1029/2000JD900384

Reply:
@RobertPincus Publication references updated to include DOIs.